.travis.yml

```diff
@@ -108,8 +109,9 @@ install:
   - pip install ".[$EXTRA_INSTALLS]" --upgrade --upgrade-strategy=eager --no-index $PRE -f file://$PWD/$WHEELDIR $VERSIONS;
 
 script:
+  # Move the data to the cache location to avoid downloading it
+  - cp -r $TRAVIS_BUILD_DIR/staticdata/* $TEST_DATA_DIR/master
```

Comment:
Turns out Travis doesn't like indented comments. This will keep it from downloading the data every time.